include/uapi/sound/sof/tokens.h

```diff
@@ -204,4 +204,18 @@
 #define SOF_TKN_AMD_ACPI2S_CH			1701
 #define SOF_TKN_AMD_ACPI2S_TDM_MODE		1702
 
+/* MODULE PIN AUDIO FORMAT */
+#define SOF_TKN_COMP_PIN_FORMAT_RATE			2000
```

Comment:
sampling rate? Unit?

Reply:
and what prevents us to add a bit more clarity moving forward? The reason why we didn't have explanations before is that the definitions were self-explanatory, but now they clearly aren't.